### PR TITLE
Update Getting Started guides with release info

### PR DESCRIPTION
**Pull request by mrjoro**
Added a section to the E2E and Quick Start Getting Started guides on how a changes gets to production and how the contributor can track where it is/test on Dev Channel.

cc @cramforce @bpaduch

### CODE REVIEW

File: contributing/getting-started-quick.md [hunk 1]
```diff
@@ -91,3 +91,10 @@ This Quick Start guide is the TL;DR version of the longer [end-to-end guide](get
 * Go to the master branch: `git checkout master`
 * Delete your local branch: `git branch -D <branch name>`
 * Delete the GitHub fork branch: `git push origin --delete <branch name>`
+
+# See your changes in production
+
+* Barring any issues releases are cut Thursday, pushed to Dev Channel immediately, pushed to 1% on Monday and pushed to all of prod on Thursday.
```
Comment (cramforce) [inline contributing/getting-started-quick.md]: 1% of AMP pages

…and pushed to all pages…

File: contributing/getting-started-e2e.md [hunk 2]
```diff
@@ -412,21 +412,34 @@ GitHub offers a convenient "Delete branch" button on the PR page after the chang
...
 ```
 
-# Celebrate
+# See your changes in production
+
+**Congratulations on making your first change to AMP!**
+
+If your change affected internal documentation, tests, the build process, etc. you can generally see your changes right after they're merged.  If your change was to the code that runs on AMP pages across the web you'll have to wait for the change to be included in a release.
+
+In general we cut a release of amphtml on Thursdays during working hours (Pacific time) and push it to the AMP Dev Channel that day.  After verifying there are no issues, we start rolling that build out the following Monday and complete the push that Thursday.
```
Comment (cramforce) [inline contributing/getting-started-e2e.md]: Maybe add: This is when all users of AMP will use your new code!

File: contributing/getting-started-e2e.md [hunk 3]
```diff
@@ -412,21 +412,34 @@ GitHub offers a convenient "Delete branch" button on the PR page after the chang
...
+
+You can see whether your change made it into the Dev Channel or production build on the [amphtml Releases page](https://github.com/ampproject/amphtml/releases).  The build marked `Draft` is the version on the Dev Channel and the build marked `Latest Release` is what is running in production.  Your Pull Request will be listed in the first build that includes it; if you don't see your Pull Request listed it will likely be in the next build.
```
Comment (cramforce) [inline contributing/getting-started-e2e.md]: It should be marked "pre-release", not Draft. Draft releases cannot be seen by non-project members.

Comment (erwinmombay) [inline contributing/getting-started-e2e.md]: it should be "the latest build marked with `prerelease` is the version on dev channel"

File: contributing/getting-started-quick.md [hunk 4]
```diff
@@ -91,3 +91,10 @@ This Quick Start guide is the TL;DR version of the longer [end-to-end guide](get
...
+* The [amphtml Releases page](https://github.com/ampproject/amphtml/releases) will list your PR in the first build that contains it.  `Draft` is the build on the Dev Channel, `Latest Release` is the build in production.
```
Comment (erwinmombay) [inline contributing/getting-started-quick.md]: update `draft` here too with "latest `prerelease`"

File: contributing/getting-started-e2e.md [hunk 5]
```diff
@@ -216,15 +216,15 @@ In the workflow we will be using you'll go to the master branch on your local re
 git checkout master
 
 # pull in the latest changes from the remote amphtml repository
-git pull upstream master 
+git pull upstream master
```
Comment (erwinmombay) [inline contributing/getting-started-e2e.md]: this assumes `upstream` is `ampproject/amphtml` right?

Comment (mrjoro) [inline contributing/getting-started-e2e.md]: Yes; for the Getting Started guides we lead people through setting up the upstream alias to that.  More advanced users can set it to whatever they want, but for the most part in these guides we try to stay away from providing too many options. :)

File: contributing/getting-started-e2e.md (hunk 2 above)
Comment (mrjoro) [inline contributing/getting-started-e2e.md]: Done.

File: contributing/getting-started-e2e.md (hunk 3 above)
Comment (mrjoro) [inline contributing/getting-started-e2e.md]: Got it; I was trying to reverse engineer the labels from where the Releases page was and what was currently on canary.  I talked to @erwinmombay about it and realized they temporarily didn't have the right labels. :)

Comment (mrjoro) [inline contributing/getting-started-e2e.md]: Done.

File: contributing/getting-started-quick.md (hunk 1 above)
Comment (mrjoro) [inline contributing/getting-started-quick.md]: Done.

File: contributing/getting-started-quick.md (hunk 4 above)
Comment (mrjoro) [inline contributing/getting-started-quick.md]: Done.

**Comment by mrjoro** [comment]
I can't merge; @erwinmombay if it looks okay can you merge when you have a chance?

**Comment by erwinmombay** [comment]
@mrjoro done

**Pull request merged**